python/mpi_cmake_modules/cpp_format.py

```diff
@@ -1,130 +0,0 @@
-"""cpp_format.py
```

Comment:
why delete this file?
Just modify it.
This is also based on clang after all...